datatypes/mcs_decimal.cpp

```diff
@@ -606,27 +606,46 @@ std::string Decimal::toStringTSInt64() const
 
 // Dispatcher method for toString() implementations
 std::string Decimal::toString(bool hasTSInt128) const
```

Comment:
OK. Here is the thing.
You add utils::NullString Decimal::toNullString(bool hasTSInt128) thus std::string Decimal::toString(bool hasTSInt128) must become obsolete and deleted, right?
I had seen that code has expressions like this:
NullString value;
...
value.assign(dec.toString(true));
See we_dmlcommandproc.cpp:3114 and there are other places.
This returns std::string to assign to NullString? NullString looks like a more appropriate type to return here.

Reply:
This requires a lot of code changes. I propose to postpone this change.